src/components/Legend.tsx

```diff
@@ -25,14 +25,26 @@ function renderType(
 }
 
 class Legend extends React.Component<LegendProps, null> {
-  renderLegendGroup(legendGroup: LegendGroup) {
+  renderLegendGroupVertical(
+    legendGroup: LegendGroup,
+    customClickBehavior?: Function
```

Comment:
Is there a more specific type signature we can use over `Function` here?

Reply:
Probably something like `d: any => void` - this is a good suggestion, I'll keep it in mind when I return to this section of the codebase.